.github/workflows/build.yml

```diff
@@ -548,7 +548,82 @@ jobs:
           export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}
           export AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}
           tar -cvf - ./container-logs | aws s3 cp - s3://${{ secrets.AWS_LOG_BUCKET_NAME }}/core-pipeline/${{ github.run_number }}/${{ github.run_attempt }}/htcmock-${{ matrix.queue }}-${{ matrix.object }}-${{ matrix.log-level }}-container-logs.tar.gz
+  testConnectivity:
+    needs:
+      - versionning
+      - buildImages
+    env:
+      VERSION: ${{ needs.versionning.outputs.version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ca: [true, false]
+    name: "Test connectivity - Ca installed ${{ matrix.ca }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+          submodules: true
+
+      - name: Install AWSCLI (the one in the Github runner does not work)
+        run: |
+          pip install awscli
+
+      - name: Setup just
+        run: |
+          sudo snap install --edge --classic just
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v2
+
+      - name: Deploy Core
+        run: |
+          just tag=${VERSION} deploy
+          sleep 10
+
+      - name: Add CA to store
+        if: ${{ matrix.ca }}
+        run: |
+         sudo apt install ca-certificates
+         sudo mkdir -p /usr/local/share/ca-certificates/
+         sudo cp terraform/.terraform/ingress_tls/server/ca.crt /usr/local/share/ca-certificates/ca_tls.crt
+         sudo cp terraform/.terraform/ingress_mtls/server/ca.crt /usr/local/share/ca-certificates/ca_mtls.crt
+         sudo update-ca-certificates
+
+      - name: Run connection test
+        run: CA_INSTALLED=${{ matrix.ca }} dotnet test Tests/Common/Client/src/ArmoniK.Core.Common.Tests.Client.csproj --filter ArmoniK.Core.Common.Tests.Client.ConnectionTest --logger "trx;LogFileName=test-results.trx"
+
+      - name: Test Report
+        uses: dorny/test-reporter@v1
+        if: success() || failure()
```

Comment:
```suggestion
        if: always()
```

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions#always
Should we ?

Reply:
I think, yes. But I don't mind keeping it this way.